### PR TITLE
Add application launcher example to constants.js

### PR DESCRIPTION
**Pull request by spadgett**

### CODE REVIEW

**Comment by dtaylor113** [comment]
LGTM.  For what it's worth, here is the example I use:
```
APP_LAUNCHER_NAVIGATION: [
    {
      title: "App One",
      href: "#/ipsum/intellegam/recteque",
      tooltip: "Launch the Function User Interface",
      iconClass: "pficon pficon-storage-domain"
    },
    {
      title: "App Two",
      href: "#/ipsum/intellegam/suavitate",
      tooltip: "Launch the Function User Interface",
      iconClass: "pficon pficon-build"
    },
    {
      title: "App Three",
      href: "#/ipsum/intellegam/lorem",
      tooltip: "Launch the Function User Interface",
      iconClass: "pficon pficon-project"
    },
    {
      title: "App Four",
      href: "#/ipsum/intellegam/home",
      tooltip: "Launch the Function User Interface",
      iconClass: "pficon pficon-home"
    }
  ]
```

File: app/scripts/constants.js [hunk 1]
```diff
@@ -446,8 +446,22 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
...
+    //   id: 1,
+    //   title: 'Microservices Application',
+    //   icon: 'fa fa-cubes',
+    //   url: 'https://www.example.com/',
+    //   description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt.'
+    // }
+  ],
+  APP_LAUNCHER_NAVIGATION: [
+    // Example application link to show in the masthead applciation launcher beside the help and logout icons.
+    // {
```
Comment (jeff-phillips-18) [inline app/scripts/constants.js]: Typo in application

Comment (spadgett) [inline app/scripts/constants.js]: Thanks, fixed

**Comment by spadgett** [comment]
[merge][severity:lowrisk]

**Comment by spadgett** [comment]
Flake #1684

[merge][severity:lowrisk]

**Pull request merged**